Lizarov,Anna.Assignment#6.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Lizarov,Anna.Assignment#6"
+author: "Anna Lizarov"
+date: "November 20, 2018"
+output: html_document
+---
+
+#Packages
+```{r}
+library(rpart)
+```
+
+#Data
+```{r}
+#Upload the data sets MOOC1.csv and MOOC2.csv
+M1 <- read.csv("MOOC1.csv", header = TRUE)
+
+M2 <- read.csv("MOOC2.csv", header = TRUE)
+
+```
+
+#Decision tree
+```{r}
+#Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
+
+#I used variables grade and assignment.  
+
```

Comment:
Good work, you could also use `forum.posts` but the increments would swamp the other two variables so you would need to scale the variables first.